ca/ca.go

```diff
@@ -28,6 +28,11 @@ type CAImpl struct {
 	log *log.Logger
 	db  *db.MemoryStore
 
+	mainChain         *chain
```

Comment:
I think that the logic in `getChain` could be simplified if there was just one `chains []*chain` field on this struct and we documented that the default chain (I think "default" is clearer than "main") is always index 0.

Reply:
I simplified it here, and also in the `Issuer` / `AlternativeIssuers` of `Certificate`.